llvm/lib/CodeGen/MIR2Vec.cpp

```diff
@@ -190,6 +197,28 @@ void MIRVocabulary::buildCanonicalOpcodeMapping() {
                     << " unique base opcodes\n");
 }
 
+MIRVocabulary MIRVocabulary::createDummyVocabForTest(const TargetInstrInfo &TII,
+                                                     unsigned Dim) {
+  assert(Dim > 0 && "Dimension must be greater than zero");
+
+  float DummyVal = 0.1f;
```

Comment:
this guy could be const static, right?

Reply:
No, I missed incrementing it in the loop. Added it now. (Tests in MIR2Vec are not looking for magic numbers like IR2Vec so it wasn't caught in tests.)